donate/templates/payment/thank_you.html

```diff
@@ -18,30 +18,41 @@
             </div>
             <h1 class="heading heading--primary heading--bottom-margin">{% trans "One last thing…" %}</h1>
             <p>{% trans "Can you help us share the important work Mozilla is doing?" %}</p>
+
             <div class="button__group">
-                <a class="button button--outline button--icon" href="https://facebook.com/mozilla">
+                {# see https://developers.facebook.com/docs/sharing/reference/feed-dialog/v2.5 #}
+                {# Add a default image to use for social sharing here in case one is not provided on the page. #}
+                <a class="button button--outline button--icon" href="https://www.facebook.com/dialog/feed?app_id={{ social_media_settings.facebook_app_id }}&amp;link=http://donate.mozilla.org&amp;picture={{ request.get_host }}{% static '_images/og.png' %}&amp;name={{ page.title|urlencode }}&amp;description=I donated to @mozilla today because I %23lovetheweb. Join me and help fight for a better and healthier internet, for all." title="Share on Facebook">
                     <svg class="button__icon button__icon--facebook" width="16" height="16">
                         <use xlink:href="#facebook"></use>
                     </svg>
                     <div class="button__label">{% trans "Facebook" %}</div>
                 </a>
-                <a class="button button--outline button--icon" href="https://twitter.com/mozilla">
+                {# see https://dev.twitter.com/web/tweet-button/web-intent #}
+                <a class="button button--outline button--icon" href="https://twitter.com/intent/tweet?text=I donated to @mozilla today because I %23lovetheweb. Join me and help fight for a better and healthier internet, for all.&amp;url=http://donate.mozilla.org&amp;via=mozilla" title="Share on Twitter">
```

Comment:
@nicklee Note that this needs to be localized *then* urlencoded (and same for FB and email)

Reply:
@TheoChevalier Updated with localisation and urlencoding

Reply:
Cheers!